.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,29 @@
+name: Continuous testing & Linting
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
```

Comment:
Python 3.5, drop?

Reply:
Ack, wasn't sure if this still needs to support 3.5 for SLES.